WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentView.swift

```diff
@@ -367,6 +367,6 @@ private extension BlazeConfirmPaymentView {
                             type: "product",
                             objective: "sales",
                             ctaText: "Shop now"),
-        image: .init(image: .iconBolt, source: .asset(asset: PHAsset())),
+        image: .init(image: .wooLogoImage()!, source: .asset(asset: PHAsset())),
```

Comment:
The `iconBolt` image was used in just a SwiftUI preview, so I replaced it with a different one, then removed it.